templates/operations.html

```diff
@@ -678,7 +678,13 @@ <h3 x-text="selectedOperation.name"></h3>
                                                 x-on:click="jitterMin = 2; jitterMax = 8;">
                                             Reset
                                         </button>
+                                        <p x-show="!isJitterValid()" class="pl-3 help has-text-danger">
+                                            Min and Max need to be >= 0, and Min must be <= Max
+                                        </p>
                                     </div>
+                                    <p x-show="!isJitterValid()" class="help has-text-danger">
+                                        Min and Max need to be >= 0, and Min must be <= Max
+                                    </p>
```

Comment:
Probably leftover from the merge conflict--lines 685-687 is a duplicate, but it doesn't have the padding so it should be removed, otherwise everything else looks good!